testing/batch_processing/test_batch_processing.py

```diff
@@ -16,6 +16,7 @@
 
 # TODO: We can and should be verifying more results produced by this pipeline, but which values?
 TESTED_VALUES = [("t2/csa_c2c3.csv", 0, "MEAN(area)"),
```

Comment:
One other neat thing: By adding the CSV file to the "`TESTED_VALUES`" list, your CSA (pmj) value now automatically shows up [in output of the batch_processing.sh script](https://github.com/spinalcordtoolbox/spinalcordtoolbox/runs/3231834684#step:4:3558), too! :smile: